app/models/enhancements/artefact.rb

```diff
@@ -36,4 +36,10 @@ def allow_specialist_sector_tag_changes?
   def allow_section_tag_changes?
     owning_app != 'publisher'
   end
+
+  def taggable?
+    return true if Settings.untaggable_apps.nil?
```

Comment:
It's minor, but the `#nil?` here would be superfluous if you used `unless Settings.untaggable_apps`